odoo/models.py

```diff
@@ -152,6 +152,11 @@ class MetaModel(api.Meta):
     """
     module_to_models = defaultdict(list)
 
+    # Deprecated methods/attributes
+    DEPRECATED_METHODS_ATTRIBUTES = [
+        'view_init',
+    ]
+
```

Comment:
Should be a set as the only operation done on this structure is `in`.

Reply:
It was :o ! 
@rco-odoo traps me .

Reply:
No it's correct as a list, because it's only being iterated, it's not checked for containment.

Checks are cheaper for containment... sometimes (mostlly for large sets, the constant is somewhat high) but for iterations they're not useful, a list takes less memory and is cheaper to iterate. The only drawback is that it doesn't clearly spell out its items are unique but that's generally not a concern for such things.

Reply:
My bad! I somehow mis-read "if attr in DEPRECATED_METHODS_ATTRIBUTES` but you are right, it is `for attr in DEPRECATED_METHODS_ATTRIBUTES: if attr in attrs:`. The list is very fine indeed!